src/VS.Web.CG.Mvc/Identity/IdentityGeneratorTemplateModelBuilder.cs

```diff
@@ -185,7 +185,7 @@ internal Type UserType
             var layout = _commandlineModel.Layout;
             if (_commandlineModel.GenerateLayout)
             {
-                layout = "/Pages/Shared/_Layout.chstml";
```

Comment:
Is this change intentional here? 
There's another PR from release/2.1 that is bringing in this change. Might lead to conflicts.